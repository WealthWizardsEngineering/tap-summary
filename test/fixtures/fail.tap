TAP version 13
# pass
ok 1 should pass
# fail
not ok 2 should fail
  ---
    operator: ok
    expected: true
    actual:   false
    at: Test.<anonymous> (/Users/zoubin/usr/src/self/tap-summary/example/error.js:9:5)
    stack: |-
      Error: should be deeply equivalent
          at Test.assert [as _assert] (/Users/ww/project/node_modules/tape/lib/test.js:314:54)
          at Test.bound [as _assert] (/Users/ww/project/node_modules/tape/lib/test.js:99:32)
          at Test.tapeDeepEqual (/Users/ww/project/node_modules/tape/lib/test.js:555:10)
          at Test.bound [as deepEquals] (/Users/ww/project/node_modules/tape/lib/test.js:99:32)
          at Test.<anonymous> (/Users/ww/project/src/env-vars.spec.js:28:5)
          at Test.bound [as _cb] (/Users/ww/project/node_modules/tape/lib/test.js:99:32)
          at Test.run (/Users/ww/project/node_modules/tape/lib/test.js:117:31)
          at Test.bound [as run] (/Users/ww/project/node_modules/tape/lib/test.js:99:32)
          at Immediate.next [as _onImmediate] (/Users/ww/project/node_modules/tape/lib/results.js:88:19)
          at processImmediate (internal/timers.js:464:21)
  ...
# pass and fail
ok 3 should pass
not ok 4 should fail
  ---
    operator: ok
    expected: true
    actual:   false
    at: Test.<anonymous> (/Users/zoubin/usr/src/self/tap-summary/example/error.js:15:5)
  ...

1..4
# tests 4
# pass  2
# fail  2

