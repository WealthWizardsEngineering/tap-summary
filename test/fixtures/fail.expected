
[33m# Tests
[0m
[G[32mâœ” pass [pass: 1, fail: 0][0m
[G[31mâœ– fail [pass: 0, fail: 1][0m
[G[31mâœ– pass and fail [pass: 1, fail: 1][0m

[33m# Summary
[0m
[36mplanned: 4[0m
[36massertions: 4[0m
[32mpass: 2[0m
[31mfail: 2[0m

[33m# Fails
[0m
[36m[4m## fail[0m[0m
[31m  âœ– should fail[0m
[36m    operator: ok
    expected: true
    actual:   false
    at: Test.<anonymous> (/Users/zoubin/usr/src/self/tap-summary/example/error.js:9:5)
    stack: |-
Error: should be deeply equivalent
at Test.assert [as _assert] (/Users/ww/project/node_modules/tape/lib/test.js:314:54)
at Test.bound [as _assert] (/Users/ww/project/node_modules/tape/lib/test.js:99:32)
at Test.tapeDeepEqual (/Users/ww/project/node_modules/tape/lib/test.js:555:10)
at Test.bound [as deepEquals] (/Users/ww/project/node_modules/tape/lib/test.js:99:32)
at Test.<anonymous> (/Users/ww/project/src/env-vars.spec.js:28:5)
at Test.bound [as _cb] (/Users/ww/project/node_modules/tape/lib/test.js:99:32)
at Test.run (/Users/ww/project/node_modules/tape/lib/test.js:117:31)
at Test.bound [as run] (/Users/ww/project/node_modules/tape/lib/test.js:99:32)
at Immediate.next [as _onImmediate] (/Users/ww/project/node_modules/tape/lib/results.js:88:19)
at processImmediate (internal/timers.js:464:21)[0m

[36m[4m## pass and fail[0m[0m
[31m  âœ– should fail[0m
[36m    operator: ok
    expected: true
    actual:   false
    at: Test.<anonymous> (/Users/zoubin/usr/src/self/tap-summary/example/error.js:15:5)[0m

